### ISSUE TITLE
"Could not start tinc" on Android 10

### ISSUE THREAD

**Issue by pacien** [pacien/tincapp#90]
The daemon doesn't start on Android 10.
Confirmed on emulator.

__Notification:__
```
Could not start tinc
tinc daemon exited during startup:
Non-zero exit status code (1).
Check the logs for more details.
```

__tinc.*.log:__
```
2019-09-13 17:22:49 tinc[6382]: fd/69 adapter set up.
2019-09-13 17:22:49 tinc[6382]: Ready
2019-09-13 17:22:49 tinc[6382]: Error while waiting for input: Bad file descriptor
2019-09-13 17:22:49 tinc[6382]: Terminating
```

**Comment by schemacs** [comment]
same here in /storage/emulated/0/Android/data/org.pacien.tincapp/cache/tincapp.log:
```
10:05:35.641 [main] INFO  o.p.tincapp.service.TincVpnService - Starting tinc daemon for network "mytinc".
10:05:36.226 [AsyncTask #2] ERROR o.p.tincapp.service.TincVpnService - tinc daemon exited during startup:
Non-zero exit status code (1).
Check the logs for more details.
org.pacien.tincapp.commands.Executor$CommandExecutionException: Non-zero exit status code (1).
	at org.pacien.tincapp.commands.Executor$forkExec$1.invoke(Executor.kt:64)
	at org.pacien.tincapp.commands.Executor$forkExec$1.invoke(Executor.kt:32)
	at org.pacien.tincapp.commands.Executor$sam$java8_util_function_Supplier$0.get(Unknown Source:2)
	at java8.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1765)
	... 3 common frames omitted
Wrapped by: java8.util.concurrent.CompletionException: org.pacien.tincapp.commands.Executor$CommandExecutionException: Non-zero exit status code (1).
	at java8.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:301)
	at java8.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java8.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1767)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
10:05:36.254 [main] INFO  o.p.t.s.ConnectivityChangeReceiver - Connectivity change intent received: Intent { act=android.net.conn.CONNECTIVITY_CHANGE flg=0x4200010 (has extras) }
10:05:36.254 [main] INFO  o.p.t.s.ConnectivityChangeReceiver - Sending immediate reconnection request to the tinc daemon.
```

**Comment by pacien** [comment]
Fixed with commit 5de0313f16382ab97c14661222d8be51c5fc6e72, release v0.28.
